### ISSUE TITLE
[DataCap Application] - Human Rights Institute WJA - Tech For Freedom

### ISSUE THREAD

**Issue by AntonioPuppio** [filecoin-project/filecoin-plus-large-datasets#1077]
# Large Dataset Notary Application
To apply for DataCap to onboard your dataset to Filecoin, please fill out the following.

## Core Information
- Organization Name: The Human Rights Institute for Peace and Freedom (HRI).
- Website / Social Media: https://forpeaceandfreedom.org/ / Twitter: @humanrightsins
- Total amount of DataCap being requested (between 500 TiB and 5 PiB): 7 PB (5 copies of 1.4 PB each). 
- Weekly allocation of DataCap requested (usually between 1-100TiB): 100 TiB
- On-chain address for first allocation: f1gzjhqbap7dklmsd3joakwwlh7vcacmjh3qdky2i

## Project details
Share a brief history of your project and organization
```
The Human Rights Institute for Peace and Freedom (HRI) was founded in Madrid (2018), as a non-profit organization with its own legal personality, autonomy, and independence of economic interests or political parties, with the objective of implementing preventive strategies and legal solutions to serious human rights violations, applying technological and digital platforms developed for defense, promotion of peace, freedom and the rule of law. The aforementioned is a human rights arm of the well-known [World Jurist Association](https://worldjurist.org/), and has a technology division named Tech For Freedom, with the mission to study the application of novel technologies as tools to promote and defend human rights. 

The HRI has been introduced to Filecoin by a company named Fungi Project, a spanish storage provider and Filecoin ambassador through The Orbit Program, who is currently trying to bring valuable datasets to the Filecoin Network.

They were extremely didactic with complex subjects such as blockchain technology and most important, the Filecoin Protocol itself. We were quite intrigued by the concept of decentralized storage, Filecoin’s vision and core mission, and how information can be stored without the control of central governments and corporations. Therefore, we decided to work with Fungi Project on the creation of a dataset integrated by institutions, media outlets, newspapers and NGOs from Venezuela, which have suffered attacks and human rights violations in the past 25 years. Our intention is the preservation of valuable and historical data for the next generations to make use of, employing the most robust technology possible. 

In order to provide some context for the Filecoin Community, Venezuela is a country in South America which has been ruled by an authoritarian regime for the past 25 years. Our country has suffered a refugee crisis with more than 7 million people having to forcibly leave, hyperinflation, permanent violations to the right of freedom of speech, censorship and much more. As a consequence, any NGO or media agency which reports against the government of Nicolas Maduro is heavily and unjustly persecuted. In parallel, the dictatorial government has operated with impunity.  Currently, the venezuelan government is sanctioned by The United Nations (UN) and more than 100 countries worldwide. 

Consequently, after conversations with several organizations, The HRI has been able to assemble a large dataset of 1.4 Petabytes named Tech For Freedom integrated by the following institutions:

- El Nacional(https://www.elnacional.com/): One of the most important and historical (79 years old) newspapers of Venezuela. The newspaper has been heavily persecuted by the government, with their headquarters even being expropriated just because of conducting the act of journalism and condemning the dictatorship. El Nacional is storing its 79 years old digitized archive on the Filecoin Network and many videos of human right violations, testimonies and interviews . [Twitter link](https://twitter.com/ElNacionalWeb). 

- El Carabobeño (https://www.el-carabobeno.com/): Also one of the most historical (89 years old) newspapers of the country. It now functions as a media outlet because of the persecution from the venezuelan government. [Twitter link](https://twitter.com/el_carabobeno). 

 - El Pitazo (https://elpitazo.net/): One of the most prominent media outlets in Venezuela and Latin America. It has won international awards for their labor of reporting in favor of free speech and human rights. El Pitazo is storing a large media archive in the Filecoin Network. [Twitter link](https://twitter.com/ElPitazoTV). 

- JEP Venezuela (https://www.jepvenezuela.com/): “Justicia, Encuentro y Perdón” is an NGO with the mission of preserving the memory of human right violations victims. JEP has done the exhaustive job of recording an abundant amount of testimonies from different sectors of the Venezuelan Society. [Twitter link](https://twitter.com/JEPvzla). 

- The Human Rights Institute for Peace and Freedom (https://forpeaceandfreedom.org/): One of our key programs has been the work of cooperating with Venezuelan Organizations and exiles to archive as many human right violations in order to work with international institutions. The HRI is also providing its stored data to this project. [Twitter link.](https://twitter.com/humanrightsins) 
```
What is the primary source of funding for this project?
```
As a Non Profit Organization, we finance our activities with donations from institutions and philanthropists. 
```
What other projects/ecosystem stakeholders is this project associated with?
```
For this particular project, Tech for Freedom is associated with Fungi Project (Storage provider and Filecoin Orbit Ambassador), and the institutions who decided to store their datasets through the project. 
```
## Use-case details
Describe the data being stored onto Filecoin
```
The dataset consists of different forms of data (mainly video) collected from 2 historical newspapers, 1 online communication agency and 2 NGOs (including ourselves).

The dataset has sensible photos and videos from street riots, testimonies from victims, interviews, protests, the historical archive of the newspapers, and more. 

```
Where was the data in this dataset sourced from?
```
It depends on each institution: 

- El Nacional: AWS and own servers. 
- El Carabobeño: AWS and own Servers. 
- El Pitazo: Microsoft Azure. 
- JEP Venezuela: AWS
- HRI: own servers. 
```
Can you share a sample of the data? A link to file would work.
```
https://ln5.sync.com/dl/cfdf05720/jdd6xsep-2hupd3vf-8532z8v5-dvms7isq

```
Confirm that this is a public dataset that can be retrieved by anyone on the Network (i.e., no specific permissions or access rights are required to view the data).
```
We confirm that this is a public dataset. 

In order to provide more transparency to the community, notaries that agree to sign an NDA will have access to the contracts signed with every organization participating in this project. 

```
What is the expected retrieval frequency for this data?
```
Files are retrievable by anyone who wants to do research about the venezuelan crisis, so it's going to be the frequency that the filecoin community demands. 

```
For how long do you plan to keep this dataset stored on Filecoin?
```
One and a half years with probable renewal. The HRI is interested in possible perpetual or permanent storage solutions. 
```
## DataCap allocation plan
In which geographies (countries, regions) do you plan on making storage deals?
```
After discussing this decision with the institutions involved, we could store the dataset in every country excluding the following: Russia, China, Iran, North Korea and Cuba given the relation between these countries' governments with the Government of Venezuela.  

The preferable regions are: North America, Europe, Asia (excluding China) and Australia. 
```
How will you be distributing your data to storage providers? Is there an offline data transfer process?
```
Currently, we are putting together the dataset in Fungi Project’s servers in Madrid (Spain). Therefore, depending on where each SP is located and their internet connection, we can host the dataset online for the SPs to download or use a physical transportation solution. 

```
How do you plan on choosing the storage providers with whom you will be making deals? This should include a plan to ensure the data is retrievable in the future both by you and others.
```
Choosing reputable providers in the community via Filecoin slack or ranking sites from the regions mentioned above. Also, we are interested in a data marketplace which the Fungi team mentioned to us named Big Data Exchange. 

Any SP interested in participating on this project could reach us at: hello@techforfreedom.org.
```
How will you be distributing deals across storage providers?
```
Hosted online (like the sample links provided) for the SPs to download or using a physical transportation solution.
```
Do you have the resources/funding to start making deals as soon as you receive DataCap? What support from the community would help you onboard onto Filecoin?
```
Since we partnered with Fungi Project, they assured us that they have the resources to start making deals as soon as we receive datacap. 
```

**Comment by raghavrmadya** [comment]
Please get the notaries who have agreed to sign an NDA to comment here. We will create a custom multisig for this application

**Comment by AntonioPuppio** [comment]
Hello @raghavrmadya! At this moment our team is preparing the NDAs for notaries to sign. When it's done, we let you know. Thanks.

**Comment by Sunnyiscoming** [comment]
Any update here?

**Comment by Sunnyiscoming** [comment]
Close for no reply.

**Comment by AntonioPuppio** [comment]
Hello @Sunnyiscoming. At this moment we are still downloading and preparing the dataset to be stored on the network, that's why we have not sent NDAs to notaries yet. By "reopen it", you mean creating a new issue, or can we use the same?
